src/Simplex_tree/include/gudhi/Simplex_tree/multi_filtrations/Box.h

```diff
@@ -0,0 +1,182 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which
+ * is released under MIT. See file LICENSE or go to
+ * https://gudhi.inria.fr/licensing/ for full license details. Author(s): David
+ * Loiseaux
+ *
+ *    Copyright (C) 2023 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+/**
+ * @file box.h
+ * @author David Loiseaux
+ * @brief BOX.
+ */
+
+#ifndef BOX_H_INCLUDED
+#define BOX_H_INCLUDED
+
+#include <gudhi/Debug_utils.h>
+#include <ostream>
+#include <vector>
+
+#include "Finitely_critical_filtrations.h"
+
+/**
+ * @brief Simple box in \f$\mathbb R^n\f$ .
+ */
+
+namespace Gudhi::multiparameter::multi_filtrations {
```

Comment:
Can you explain how you choose what goes into `Gudhi::multiparameter::multi_filtrations` vs what goes directly in `Gudhi::multiparameter`?